src/pages/my.tsx

```diff
@@ -12,18 +10,14 @@ const scaffoldOptions = {
 };
 
 export const getServerSideProps : GetServerSideProps = scaffold(async (ctx) => {
-    const queryClient = new QueryClient();
-
-    await queryClient.prefetchQuery('userActionResponses', async () => {
+    await ctx.queryClient.prefetchQuery('userActionResponses', async () => {
         const res = await ctx.apiFetch('/users/me/action_responses');
         const data = await res.json();
         return data;
     });
 
     return {
-        props: {
-            dehydratedState: dehydrate(queryClient),
-        },
+        props: {},
```

Comment:
It should be possible to return null (or void) here, and not have to create a props object at all.